lib/machinery_helper.rb

```diff
@@ -34,33 +31,42 @@ def initialize(s)
     @system = s
     @arch = @system.arch
 
-    @local_helpers_path = "/usr/share/machinery/helpers"
+    @local_helpers_path = File.join(Machinery::ROOT, "machinery-helper")
+    @remote_helper_path = File.join(Machinery::REMOTE_HELPER_PATH, "machinery-helper")
```

Comment:
There should be methods to access both paths and they should be used at all other places in this file instead of accessing the member variable directly.